ui/components/directory/layout/ListingHomepageLayout.tsx

```
@@ -0,0 +1,37 @@
+import type { FC } from "react";
+import DirectoryHeader from "./DirectoryHeader";
+import HeroSearchSection from "../hero/HeroSearchSection";
+import CategoryGrid from "../categories/CategoryGrid";
+import QuickFiltersBar from "./QuickFiltersBar";
+import { mockCategories, mockStats, mockFilterOptions, mockTrendingBusinesses, mockNewBusinesses, mockVerifiedBusinesses, mockCurrentLocation } from "../mockData";
+import LocationBanner from "../shared/LocationBanner";
+import TrendingSection from "../business/TrendingSection";
+import NewBusinessesShowcase from "../business/NewBusinessesShowcase";
+import VerifiedCarousel from "../business/VerifiedCarousel";
+
+export interface ListingHomepageLayoutProps {}
```

Biome (2.1.2)

[error] An empty interface is equivalent to {}.

Safe fix: Use a type alias instead.


(lint/suspicious/noEmptyInterface)